diff -rupN a2jmidid-9.old/a2j a2jmidid-9/a2j
--- a2jmidid-9.old/a2j	2019-09-24 00:34:12.000000000 +0200
+++ a2jmidid-9/a2j	2026-03-14 21:36:27.322341402 +0100
@@ -7,11 +7,11 @@
 if test $# -eq 1 -a x$1 = x-e
 then
     echo "hardware ports export"
-    a2j_control ehw
+    a2j_control --ehw
 else
-    a2j_control dhw
+    a2j_control --dhw
 fi
 
-a2j_control start
-trap "a2j_control stop; exit" INT TERM
+a2j_control --start
+trap "a2j_control --stop; exit" INT TERM
 read unused
